src/04-modality-specific-files/03-electroencephalography.md

```diff
@@ -354,12 +405,30 @@ head.
 
 General fields:
 
+{{ MACROS___make_metadata_table(
+   {
+      "IntendedFor": (
+         "OPTIONAL",
+         "This identifies the MRI or CT scan associated with the electrodes, "
+         "landmarks, and fiducials.",
+      )
```

Comment:
What does everyone think of this approach for extending descriptions for modality- or table-specific purposes?